unitex/src/fr/umlv/unitex/frames/GraphPathFrame.java

```diff
@@ -133,9 +133,9 @@ private JPanel constructTopPanel() {
 		final JPanel top1 = new JPanel(new FlowLayout(FlowLayout.LEFT));
 		final ButtonGroup pathWithSubGraph = new ButtonGroup();
 		
-		onlyPaths = new JRadioButton("Only paths", true);
+		onlyPaths = new JRadioButton("Explore recursively", true);
```

Comment:
The better would be to prepend a label "Calls to subgraphs:" before the the two option buttons, something like this:

`Calls to subgraphs: (o) Explore calls recursively ()  Print calls without exploring it`

This is consistent with the stated in the User Manual, section 6.5, Exploring grammar paths.

Reply:
Do you also want me to rename the choices ?

Reply:
Yea is that, `Explore calls recursively` for `Only paths` and `Print calls without exploring it` for  `Do not explore subgraphs recursively`. The visual result would be something like this:

`Calls to subgraphs: (o) Explore calls recursively () Print calls without exploring it`

Thank your for your help !

Reply:
We came up with a new solution we thought was more clear with @eric-laporte.
It looks like this: 

![explore_graph_ui_01](https://user-images.githubusercontent.com/12595039/40618218-2125f414-6291-11e8-8bec-609561a22b8d.png)